### ISSUE TITLE
Is the ATLMFC issue submitted upstream?

### ISSUE THREAD

**Issue by BillyONeal** [WinampDesktop/winamp#103]
https://github.com/WinampDesktop/winamp/blob/e58d83b4a841f37649ca28070313a7b00160b480/README.md#L44-L47

Why does the `::` need to be removed in the first place?

**Comment by BillyONeal** [comment]
(Also sorry because it was probably me who added the `::` :/)

**Comment by shinigamichan** [comment]
The "fix" isn't a fix and will just recurse infinitely if the line of code is reached https://github.com/WinampDesktop/winamp/blob/e58d83b4a841f37649ca28070313a7b00160b480/Src/Plugins/Library/ml_plg/atltransactionmanager.h#L427

I don't know why somebody thought they needed this "fix" but a cursory search of the codebase shows a suspect https://github.com/WinampDesktop/winamp/blob/e58d83b4a841f37649ca28070313a7b00160b480/Src/gracenote/gracenote.h#L7

Protip: Don't edit system headers you don't understand just because you got a compiler error. Find the root cause. 

TLDR: Don't pass go, don't report upstream. Run very far from this mess of a codebase.
